sync-files/anemoi/all/.github/CODEOWNERS

```diff
@@ -1,6 +1,7 @@
-# CODEOWNERS file
+# Workflows
+/.github/ @ecmwf/AnemoiSecurity
 
-# Protect workflow files
-/.github/ @theissenhelen @jesperdramsch @gmertes @b8raoult @floriankrb @anaprietonem @HCookie @JPXKQX @mchantry
-/.pre-commit-config.yaml @theissenhelen @jesperdramsch @gmertes @b8raoult @floriankrb @anaprietonem @HCookie @JPXKQX @mchantry
-/pyproject.toml @theissenhelen @jesperdramsch @gmertes @b8raoult @floriankrb @anaprietonem @HCookie @JPXKQX @mchantry
+# Project configs
+/pyproject.toml @ecmwf/AnemoiSecurity
```

Comment:
As discussed this file is valid and the 'warning/error' is just to do with the fact that `AnemoiSecurity` does not have write access to the reusable-workflows repo, so we can ignore that error